verify.js

```diff
@@ -57,15 +64,15 @@ class Verify {
         .then(res => {
           if (res.status !== 200) {
             debug(`${buildId} | Fail ${uri}: ${res.status}`);
-            return next(null, uri);
+            return next(null, { buildId, uri, reason: `status ${res.status}` });
```

Comment:
Maybe more human readable as 
```suggestion
            return next(null, { buildId, uri, reason: `Received HTTP status ${res.status}` });
```